context/app/static/js/components/detailPage/multi-assay/ComponentAlert/ComponentAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import Typography from '@mui/material/Typography';
+import Skeleton from '@mui/material/Skeleton';
+import Button from '@mui/material/Button';
+
+import { DetailPageAlert } from 'js/components/detailPage/style';
+import { InternalLink } from 'js/shared-styles/Links';
+import { Stack } from '@mui/material';
+import useRelatedMultiAssayDatasets from '../useRelatedMultiAssayDatasets';
+
+function ComponentAlert() {
+  const { datasets, isLoading } = useRelatedMultiAssayDatasets();
+
+  const primary = datasets.raw[0] ?? {};
+
+  if (!isLoading && !primary) {
+    return null;
+  }
+
+  return (
+    <DetailPageAlert
+      severity="info"
+      sx={{
+        'div:nth-child(2)': {
+          flexGrow: 1,
+        },
+      }}
```

Comment:
This targets an otherwise inaccessible element in the alert component. We could potentially use a class selector, but the classes have those hash suffixes to avoid namespace collisions.

Reply:
Ah, gotcha; that makes sense - so these classes aren't targetable? https://mui.com/material-ui/api/alert/#classes

The `.MuiAlert-message` class seems potentially usable here but I could be mistaken, CSS-in-JS is a bit fiddly in these cases

In any case - if that doesn't work, then we can definitely just use the existing solution 😄